server/file_manager/file_manager.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package file_manager;
+
+service FileManager {
+    rpc PutFile(PutFileRequest) returns (PutFileResponse);
+    rpc GetFile(GetFileRequest) returns (GetFileResponse);
+}
+
+message PutFileRequest {
+    string fileName = 1;
+    string fileContext = 2;
```

Comment:
Good. The field ID of new fields must be monotonically incremented. And, field IDs shouldn't be reused/reassigned for backward compatibility.